schema/shared/src/main/scala/zio/blocks/schema/derive/DerivationBuilder.scala

```diff
@@ -54,17 +54,8 @@ final case class DerivationBuilder[TC[_], A](
         acc + (override_.optic -> acc.getOrElse(override_.optic, Vector.empty).appended(override_.modifier))
     }
 
-    def extraModifiers(reflect: Reflect.Type, optic: DynamicOptic): Vector[reflect.ModifierType] = {
-      val extra = modifierMap.getOrElse(optic, Vector.empty)
-      (reflect match {
-        case Reflect.Type.Record         => extra.collect { case m: Modifier.Record => m }
-        case _: Reflect.Type.Sequence[_] => extra.collect { case m: Modifier.Seq => m }
-        case _: Reflect.Type.Map[_]      => extra.collect { case m: Modifier.Map => m }
-        case Reflect.Type.Dynamic        => extra.collect { case m: Modifier.Dynamic => m }
-        case Reflect.Type.Primitive      => extra.collect { case m: Modifier.Primitive => m }
-        case _                           => Vector.empty
-      }).asInstanceOf[Vector[reflect.ModifierType]]
-    }
+    def extraModifiers(optic: DynamicOptic) =
+      modifierMap.getOrElse(optic, Vector.empty).collect { case m: Modifier.Reflect => m }
```

Comment:
Ah, I just noticed we can't override term modifiers?

Reply:
Currently, yes, we cannot override term modifiers.

I will try to add `DerivationBuilder.instance` and `DerivationBuilder.modifier` methods in follow up PRs

Reply:
I have `DerivationBuilder.modifier` almost ready, let me open a quick PR